syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetAttributeValueNameRequest {
    uint32 symptom_id = 1;
    uint32 attribute_id = 2;
    uint32 value_id = 3;
}

message GetAttributeValueNameResponse {
    string attribute_value_name = 1;	
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}